src/db_queries/admin_partners_mentors.ts

```diff
@@ -45,7 +44,14 @@ export class AdminPartnersMentors {
     searchParameters: PartnerMentorsSearch,
     client: pg.PoolClient
   ): Promise<Array<PartnerMentor>> {
-    const { searchString, courseFromDate, courseToDate, searchByName, searchByEmail, searchByStudent, searchByStudentOrganization }: PartnerMentorsSearch = searchParameters;
+    const { searchString, courseFromDate, courseToDate }: PartnerMentorsSearch =
+      searchParameters;
+    const searchByName = searchParameters.searchByName === "true";
```

Comment:
Can't we use booleans instead of strings?

Reply:
in case of search-params e.g `?searchByName=true` all values would be string